packages/svg-serializer/index.js

```diff
@@ -1,66 +1,134 @@
-const {CSG} = require('@jscad/csg')
+/*
```

Comment:
did you check the outputs of this serializer ? it was very finicky last time I worked on, and very easy to break :(

Reply:
We have a extensive library of examples from WWW and others. But I'm not really interested in testing this extensively here. Maybe we can add some more specific tests once CSG V2 is available.